tests/test_utils.py

```diff
@@ -0,0 +1,111 @@
+                                'iqr', 'chest_computed_tomographic_scans', 'non-invasive_ventilation',
+                                'patient-to-patient']
+
+        expected_study_ngram_map = {'cell culture': 'cell_culture',
+                                    'fetal bovine serum': 'fetal_bovine_serum',
+                                    'air-liquid interface': 'air-liquid_interface',
+                                    'affected health professionals': 'affected_health_professionals',
+                                    'chest computed tomographic scans': 'chest_computed_tomographic_scans',
+                                    'non-invasive ventilation': 'non-invasive_ventilation'}
+
+        self.assertTrue(list_compare(study_names, expected_study_names))
+        self.assertEqual(study_ngram_map, expected_study_ngram_map)
+
+    def test_make_ngram_map_and_list_treatment_words(self):
+        """Test make_ngram_map_and_list on treatment words."""
+        # TODO
+        # treatment_names_path = Path('test_files/sample_treatment_words.txt')
+        pass
+
+    def test_make_ngram_map_and_list_virus_words(self):
+        """Test make_ngram_map_and_list on virus words."""
+        # TODO
+        # virus_names_path = Path('test_files/sample_virus_words.txt')
+        pass
+
+    def test_pull_mentions_lemmatized(self):
+        """Test pull_mentions_lemmatized."""
+        # TODO
+        path = [PDF_JSON_DOCUMENT_PARSES]
+        lemmatized_paper = pull_mentions_lemmatized(path, title_blocks, 'title', self.expected_virus_words)
+        self.assertTrue(lemmatized_paper is not None)
+
+    def test_pull_mentions_direct(self):
+        """Test pull_mentions_direct."""
+        # TODO
+        path = [PDF_JSON_DOCUMENT_PARSES]
+        lemmatized_paper = pull_mentions_direct(path, title_blocks, 'title', self.expected_virus_words)
+        self.assertTrue(lemmatized_paper is not None)
+
+    def test_run_replace(self):
+        """Test run_replace."""
+        # TODO super basic test because anticipating this function to be changed
+        s = 'the quick brown fox jumped over the lazy dog.'
+        replace_dict = {'the quick': 'the_quick', 'fox jumped': 'fox_jumped'}
+        expected_replaced_s = 'the_quick brown fox_jumped over the lazy dog.'
+        self.assertEqual(run_replace(s, replace_dict), expected_replaced_s)
+
+    def test_extract_to_csv(self):
+        """Test extract_to_csv."""
+        # TODO
+        pass
+
+    def test_same_sentence_check(self):
+        """Test sentence enders are properly detected."""
+        block = get_text(SAMPLE_SENTENCE)
+        self.assertTrue(same_sentence_check(block, 0, 98))
+        self.assertFalse(same_sentence_check(block, 166, 168))
+
+    def test_convert_date_to_year(self):
+        """Test date is properly converted."""
+        # TODO there might be a more robust way to write this with try except statements.
+        pass
+
+
+if __name__ == '__main__':
```

Comment:
Don't add this `if __name__ == '__main__'` section at all - running the tests should be the job of `tox`